### ISSUE TITLE
redir in AWS

### ISSUE THREAD

**Issue by boussou** [troglobit/redir#14]
impossible to make it work in Amazon EC2.

As you might know, in AWS there is a local private IP (ie 10.0.0.x) mapped (routed) to a public IP.
I tried with --laddr providing local private IP
I tried  --bind with public IP
I obvisouly opened the port in Security Group

It never  receives the connection

**Comment by troglobit** [comment]
Hi, glad to hear you are trying out redir!  I've never used AWS, but I guess we're still talking IP here, so my limited knowledge of the matter is hopefully good enough. 

To help you I'd like to ask a few questions:

- What version of redir are you using, you seem to use the old (pre-github) syntax?
- Could you please share the complete command line you've tried? 

All the best
/Joachim

**Comment by boussou** [comment]
Actually I am using it  often  for smthing like 3 years ;-)
You were right, it is just IP & TCP.

If you don't know, AWS maps a public IP to an private IP dynamically (yes, NAT ;-)
There is on top of that some firewall features,  a way to apply filters like IPtables.  

I finally found out, it was just that I was on WIFI on a network controlled by a captive portal. And I was trying to connect to Mysql: it's just that the port was filtered by the source network, not the Amazon's.

I just switched redir to port 8080 - problem solved.
